### ISSUE TITLE
Rename code artifacts and introduce a !topicrole alias for !pingrole

### ISSUE THREAD

**Issue by nathan-alden-sr** [discord-csharp/MODiX#628]
https://github.com/discord-csharp/MODiX/pull/627

This PR was really just a change to the UI to make the module more discoverable, but optimally classes and interfaces would be renamed to reflect the new canonical name. A bigger question is also whether to add a `!topicrole` alias (we could keep `!pingrole` for backward compatibility if desired, or remove it, if desired).

I'm a new contributor to MODiX so please be gentle. :)

**Comment by Scott-Caldwell** [comment]
It definitely seems preferable IMO not to have 3 separate names for the same thing. As it stands now, "ping role", "marker role", and "topic role" are all used inconsistently throughout the feature, and I have some concerns that this may lead to confusion.

If we've decided that "topic roles" is the official name for this feature, then I think it makes sense to update all "ping role" and "marker role" references to "topic role".

Potentially, that could involve a migration for the designated role type as well if we cared enough to go that far, but I'm less concerned about that than the command text and messaging.
